src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  max-width: 1199px;
```

Comment:
remove max-width, snake should occupty 100% of the viewport's space

<img width="1439" alt="Screenshot 2022-06-30 at 15 23 02" src="https://user-images.githubusercontent.com/57598825/176676066-71e72325-1401-4142-b1ea-d69d28f84aaa.png">


```suggestion
```